src/components/ProfileForm/DogForm/dogForm.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from "react";
+import Doginfodetails from "./doginfodetails";
+import { Button, Form, Card } from "react-bootstrap";
+import UserForm from "../userform";
+class DogForm extends Component {
+  state = {
+    step: 1,
+    dogs: this.props.dogs
+  };
+  nextStep = () => {
+    const { step } = this.state;
+    this.setState({
+      step: step + 1
+    });
+  };
+  prevStep = () => {
+    const { step } = this.state;
+    this.setState({
+      step: step - 1
+    });
+  };
+
+  handleChange = dogs => {
+    this.setState({ dogs: dogs });
+    this.props.handleDogChange(dogs);
+  };
+
+  showStep = () => {
+    const { step, dogs } = this.state;
+    if (step === 1)
+      return (
+        <Doginfodetails
+          handleChange={this.handleChange}
+          nextStep={this.nextStep}
+          prevStep={this.prevStep}
+          passDogsToParent={this.callbackFunction}
+          dogs={dogs}
+        />
+      );
+
+    if (step === 0)
+      return (
+        // console.log("updated dogform dogs", this.state.dogs)
+        <UserForm dogs={dogs}/>
+      );
+  };
+
+  // handleChange = e => {
+  //   if (["dogName", "age", "breed"].includes(e.target.className)) {
+  //     let dogs = [...this.props.dogs];
```

Comment:
Do we need this commented out code?

Reply:
This whole component is outdated, we don't use that anymore. Deleted